let response = receiveFirst().then((r)=>{
    // success of first
    receivedSecond().then((r1)=>{
        // second call success
    })
})

// USe Seperate Promise Calls
p1
p2
p3
// Use Promise.all(), 10 overloads
// Promise.all(T1)
// Promise.all(T1,,,,,,,,T10)
    - Promise divide the execution into Micro Tasks



 # Exception Management in React-Component
1. Use try..catch block in render() of the 'Class-Component' to manage the execption and render the fallback UI 

# React 16.8+
-  Functional Components with Hooks
    - Say bye bye to Class Components    


